js/editTask.js

```diff
@@ -0,0 +1,44 @@
+import { EDIT, COMPLETE, DELETE_TODO } from "/js/constants.js";
+import handleSaveButtonClick from "/js/utility/SaveBtnAction.js";
+import handleDeleteButtonClick from "/js/utility/DltBtnAction.js";
+import handleCheckButtonClick from "/js/utility/ChkBtnAction.js";
```

Comment:
Let's be uniform about naming variables and file names.

```suggestion
import handleSaveButtonClick from "/js/utility/SaveButtonAction.js";
import handleDeleteButtonClick from "/js/utility/DeleteButtonAction.js";
import handleCheckButtonClick from "/js/utility/CheckButtonAction.js";
```

Reply:
done!